plugins/inventory_sync/module/inventory_sync.lua

```diff
@@ -56,6 +56,19 @@ function inventory_sync.serialize_player(player, player_record)
 		return
 	end
 
+	-- mod-compat: Exit Space Exploration satelite view to restore the real character
+	if remote.interfaces["space-exploration"] then
+		local status, result = pcall(function()
+			if remote.call("space-exploration", "remote_view_is_active", { player = player }) then
+				remote.call("space-exploration", "remote_view_stop", { player = player })
+			end
+		end)
+		if not status then
+			log("ERROR: Exiting remote view failed for " .. player.name .. ": " .. result)
+			player.print("ERROR: Exiting remote view failed: " .. result)
```

Comment:
If this mode is similar to editor mode (as in it is not support due to the empty inventory) then should this not also call `error` as above?

Reply:
Consider what would happen if the the remote view mode was removed entirely from Space Exploration, remote calls included. If this called `error` then instead of printing an error to the console you would no longer be able to use inventory sync at all. If it fails to switch the mode it should still try to sync it as the alternative is worse.

This is different from the editor mode where if it fails to toggle it off there's no way to access the character inventory, no way to sync the data.